roles/1_prepare/tasks/check-requirements.yml

```diff
@@ -9,7 +9,8 @@
         (ansible_distribution == 'Ubuntu' and ansible_distribution_release == 'focal') or
         (ansible_os_family == 'RedHat' and ansible_distribution_major_version == '6') or
         (ansible_os_family == 'RedHat' and ansible_distribution_major_version == '7') or
-        (ansible_os_family == 'RedHat' and ansible_distribution_major_version == '8')
+        (ansible_os_family == 'RedHat' and ansible_distribution_major_version == '8') or
+        (ansible_os_family == 'AlmaLinux' and ansible_distribution_major_version == '8')
     msg: >-
       The only supported platforms for this release are Debian Stretch, Ubuntu 18.04 LTS (Bionic) or
```

Comment:
can you edit the message also to mention centos 8 and alma? we seem to have missed it

Reply:
Done.